WordPressUI/WordPressUI/Extensions/UIImageView+Networking.swift

```diff
@@ -0,0 +1,122 @@
+import Foundation
+
+
+public extension UIImageView {
+
+    /// Downloads an image and updates the current UIImageView Instance.
+    ///
+    /// - Parameters:
+    ///     -   url: The target Image's URL.
+    ///     -   placeholderImage: Image to be displayed while the actual asset gets downloaded.
+    ///     -   pointSize: *Maximum* allowed size. if the actual asset exceeds this size, we'll shrink it down.
+    ///
+    public func downloadResizedImage(at url: URL?, placeholderImage: UIImage? = nil, pointSize: CGSize) {
```

Comment:
`at:` as a parameter name feels a bit awkward to me — I don't think I'd say "I'm downloading a picture at URL: foo.bar/xyz". 

"I'm downloading an image `from` URL" seems more natural to me. WDYT?